modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CP2ActivationKeys.macro

```diff
@@ -99,4 +99,12 @@ definition {
 		AssertVisible(locator1 = "CP2ActivationKeys#TOOLTIPS_TEXT");
 	}
 
+	macro assertDateFormat {
+		var key_linePosition = "${linePosition}";
+		var key_colunmPosition = "5";
+		AssertTextEquals(
+			locator1 = "CP2ActivationKeys#TABLE_ELEMENT",
+			value1 = "${date}");
+	}
+
```

Comment:
Is there a way for us to retrieve it without saying the columnPosition? Is there a chance of this table to increase columns and the date moves from "5" to "6", for example?